html-block-and-inline/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Block and Inline
+    </title>
+  </head>
+  <body>
+    <h1>
+      Blue Americana Snuggie&reg; Blanket
+    </h1>
+    <h2>
+      by SnuggieStore
+    </h2>
+    <p>
+      Price: <del>$19.99</del> $14.99
+    </p>
+    <p>
+      Part of our new Americana collection, this <em>beautiful</em> blue is already a customer favorite!
+    </p>
+    <p>
+      Enjoy complete <b>Freedom of Movement!</b>
```

Comment:
If you want **Freedom of Movement** to be read stronger by screen readers, I'd suggest using `<strong>` instead of `<b>`.

Reply:
sounds good!